app/helpers/application_helper.rb

```diff
@@ -397,7 +397,20 @@ def admin_links_for(community)
         :icon_class => icon_class("topbar_menu"),
         :path => admin_topbar_edit_path,
         :name => "topbar"
-      },
+      }
+    ]
+
+    if APP_CONFIG.display_knowledge_base_articles
```

Comment:
This is confusing. Whether or not we want to display knowledge base articles should toggle the landing page view. Couple ideas how to fix:

1. Do we need to conditionally show the page? Could we show it for everyone? Can we change the content so that we could show it to everyone?
2. Add a new environment variable, something like `show_landing_page_admin`